lua/tabnine/config.lua

```diff
@@ -7,7 +7,8 @@ function M.set_config(o)
 		accept_keymap = "<Tab>",
 		dismiss_keymap = "<C-]>",
 		debounce_ms = 800,
-		suggestion_color = { gui = "#808080", cterm = 244 },
```

Comment:
I don't love that we're changing the defaults here, but if we want to prioritize `suggestion_color` over `suggestion_hl_group`, then I suppose this is necessary